WcaOnRails/app/models/competition.rb

```diff
@@ -407,16 +430,8 @@ def latitude_radians
     end
   end
 
-  def events
-    # See https://github.com/cubing/worldcubeassociation.org/issues/95 for
-    # what these equal signs are about.
-    (eventSpecs || []).split.map { |e| Event.find_by_id(e.split("=")[0]) }.sort_by &:rank
-  end
-
   def has_events_with_ids?(event_ids)
-    # See https://github.com/cubing/worldcubeassociation.org/issues/95 for
-    # what these equal signs are about.
-    (event_ids - eventSpecs.split.map { |e| e.split("=")[0] }).empty?
+    (event_ids - events.ids).empty?
```

Comment:
Hmmm, not in this in particular case since it's only used in the controller:

`@competitions = @competitions.select { |competition| competition.has_events_with_ids?(params[:event_ids]) }`

It's only if we called this procedure somewhere during the form cycle. So based on that should we keep it as it is, or update it in case it's ever used in such a scenario?